src/components/Routes/Routes.js

```diff
@@ -7,6 +7,7 @@ import ProductDetailsPage from '../../containers/ProductDetailsPage';
 import ProductListPage from '../ProductListPage';
 import NotFoundPage from '../NotFoundPage';
 import DefaultLayout from '../DefaultLayout';
+import WhatIsNew from '../WhatIsNew/WhatIsNew';
```

Comment:
If you have index file inside the folder so you can import just `import WhatIsNew from '../WhatIsNew` without last part `/WhatIsNew`

Reply:
Forgot to change paths )